webapp/channels/src/components/admin_console/dropdown_setting.tsx

```diff
@@ -2,11 +2,22 @@
 // See LICENSE.txt for license information.
 
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, {Component, ReactNode, ChangeEvent} from 'react';
 
 import Setting from './setting';
 
-export default class DropdownSetting extends React.PureComponent {
+type Props = {
+    id: string;
+    values: Array<{text: string; value: string}>;
+    label: ReactNode;
+    value: string;
+    onChange: (id: string, value: string) => void;
+    disabled?: boolean;
+    setByEnv: boolean;
+    helpText: ReactNode;
+}
+
+export default class DropdownSetting extends Component<Props> {
```

Comment:
class DropdownSetting extends React.PureComponent

It was pure components

Reply:
Thank you for the review. Updated.